menon2021re/journal/metadata.yaml

```diff
@@ -55,8 +40,8 @@ keywords: rescience c, rescience x
 # Software Heritage.
 #   see https://guides.github.com/activities/citable-code/
 code:
-  - url: https://github.com/rescience-c/template
-  - doi: 
+  - url: https:/github.com/alchemi5t/MNADrc
+  - doi: 10.5281/zenodo.4681515
   - swh: 
```

Comment:
Please add the Software Heritage Foundation URI for your code (instructions in README)